CMakeLists.txt

```diff
@@ -171,6 +171,14 @@ opm_add_test(flow_blackoil_dunecpr
   DEPENDS "opmsimulators"
   LIBRARIES "opmsimulators")
 
+
+opm_add_test(flow_blackoil_dunecpr_noflex
+  ONLY_COMPILE
+  DEFAULT_ENABLE_IF ${FLOW_DEFAULT_ENABLE_IF}
+  SOURCES flow/flow_blackoil_dunecpr_noflex.cpp
+  EXE_NAME flow_blackoil_dunecpr_noflex
+  CONDITION NOT MPI_FOUND)
```

Comment:
I have the feeling that some lines went missing during rebasing or merging. Shouldn' there be something like
```
DEPENDS "opmsimulators"
  LIBRARIES "opmsimulators"
```

I am also not 100% sure what `CONDITION NOT MPI_FOUND` means. Does it mean cpr is only available if MPI is not found? That would be a bummer.